ajax/libs/RecordRTC/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "RecordRTC",
+  "filename": "RecordRTC.min.js",
+  "description": "RecordRTC is a server-less (entire client-side) JavaScript library can be used to record WebRTC audio/video media streams. It supports cross-browser audio/video recording.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/muaz-khan/RecordRTC.git"
+  },
+  "homepage": "http://RecordRTC.org/",
+  "keywords": [
+    "webrtc",
+    "recordrtc",
+    "audio-recording",
+    "video-recording",
+    "gif-recording",
+    "audio/video recording"
+  ],
+  "author": {
+    "name": "Muaz Khan",
+    "email": "muazkh@gmail.com",
+    "url": "http://www.muazkhan.com/"
+  },
+  "license": "MIT",
+  "npmName": "recordrtc",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "RecordRTC*"
```

Comment:
@ParkourKarthik You did a great work.
Let's modify from `RecordRTC*` to `RecordRTC*.+(js|css|map)`.
Because the current pattern will get the RecordRTC.md file which we don't need it.
Thank you.